pkg/openstack/loadbalancer.go

```diff
@@ -1549,6 +1567,10 @@ func (lbaas *LbaasV2) checkService(service *corev1.Service, nodes []*corev1.Node
 		return fmt.Errorf("no service ports provided")
 	}
 
+	if len(service.Spec.IPFamilies) > 0 {
```

Comment:
I assume this should doc somewhere as it's a logic that embeded in the code
first service IP family will be used?

Reply:
Agree, let me add some comments and maybe also a dual-stack section into `docs/openstack-cloud-controller-manager)/expose-applications-using-loadbalancer-type-service.md`

Reply:
done